LondonBritishLibrary/add/BLadd19960.xml

```diff
@@ -0,0 +1,220 @@
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.7">
+                                <locus from="126r"/>
+                                <title type="complete" ref="LIT3108RepCh388"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.8">
+                                <locus from="128r"/>
+                                <title type="complete" ref="LIT2731RepCh15"/>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">መኃልይ፡ ለእግዚአብሔር፡ ዐቢይ። በዕዝል፡ ፫፡ ጊዜ፡ ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ። ለአዳም፡ አቡነ፡ ዘለሐኮ፡ በአርአያከ።
+                                እግዚአብሔር፡ ርድአነ፡ መርዔተከ። ሃሌ፡ ሉያ፡ ሃ፡ ሃ፡ ለአሌል፡ ቅቱል፡ ዘነጸርከ፡ ቍርባኖ። ይእዜኒ፡ ስምዓነ፡ ለአድኅኖ።</incipit>
+                            </msItem>
+                        </msItem>
+                        
+                        
+                        
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>   
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">134</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>3.88</height>
+                                        <width>3.25</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="deficient">Much stained and slightly mutilated.</condition>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="28 29"/>
+                                
+                                <layout columns="2" writtenLines="28 29">
+                                    <locus from="114" to="130"/>
+                                </layout>
+                            </layoutDesc>
+
+                        </objectDesc>
+
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <date notBefore="1600" notAfter="1699"/>
+                                <desc>Written in a neat hand.</desc>
+                            </handNote>
+                            <handNote script="Ethiopic" xml:id="h2" corresp="#a2">
+                                <date notBefore="1699" notAfter="1877"/>
+                                <desc>Written in a later hand.</desc>
```

Comment:
and the hand of a1?

Reply:
I believe that many years ago we decided not to add hands for additions if we have no information on them, because it was assumed that an addition is by definition in another than the main hand. My principle was to only add a hand for an addition if the catalogue gives information on it (such as here h2).

Reply:
I will merge this now, since it is already approved, and the hand issue is larger than this singe PR. If you would like to start a discussion or think that I misremember something, please open an issue. THank you!